templates/students/index.html

```diff
@@ -0,0 +1,85 @@
+{% extends 'layout.html' %} 
+
+{% load static %} 
+
+{% block title %} Jagrati | Students {% endblock %} 
+
+{% block style %}
+<link rel="stylesheet" type="text/css" href="{% static 'students/css/style.css' %}" />
+{% endblock %} 
+
+{% block content %}
+
+
+<div class="vol__header">
+  <div class="fixed-bg">
+    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+      <ol class="carousel-indicators">
+        <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+        <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+        <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+      </ol>
+      <div class="carousel-inner">
+        <div class="carousel-item active">
+          <img class="d-block w-100" src="{% static 'home/images/02.jpg' %}" alt="First slide" style="height:50vh;">
```

Comment:
I don't think we should keep the make the carousel cover half of the screen. Maybe keep it to just 100-150 px, whatever height looks good. You may also add some media queries for smaller screen sizes.

Reply:
sir for now I have removed the carousel...If in future needed...then we will work on that sir...